ibm/acctest/acctest.go

```diff
@@ -637,6 +638,12 @@ func init() {
 		fmt.Println("[INFO] Set the environment variable PI_PVM_INSTANCE_ID for testing Pi_instance_name resource else it is set to default value 'terraform-test-power'")
 	}
 
+	Pi_deployment_type = os.Getenv("PI_DEPLOYMENT_TYPE")
+	if Pi_deployment_type == "" {
+		Pi_deployment_type = "terraform-test-power"
+		fmt.Println("[INFO] Set the environment variable PI_DEPLOYMENT_TYPE for testing Pi_deployment_type resource else it is set to default value 'terraform-test-power'")
```

Comment:
Can we use `PI_INSTANCE_DEPLOYMENT_TYPE`?
Also, change the info message "...for testing ibm_pi_instance resource..."